### ISSUE TITLE
Wrong return

### ISSUE THREAD

**Issue by carmolim** [aristath/ariColor#1]
First, nice plugin! I was looking for something like this for a while.

I'm having a problem with your example to generate a darker color, I created a function: 

```
function category_color_brightness( $color, $amount ) {

    // Create a new object using hex as our original color
    $color_obj = ariColor::newColor( $color, 'hex' );

    // Darken the color by some amount
    $result_color = $color_obj->getNew( 'lightness', $color_obj->lightness - $amount );

    // return HEX color
    return $result_color->hex;
}
```

The input  `$color` is: `'#505050'` but it always returns: `#0-3700-3700-370`.

I tried with the `$result_color->toCSS( 'hex' );` but it returns the same thing.

What I'm missing?

**Comment by aristath** [comment]
Hey there!

I'm unable to replicate your issue...
I just copied your function in my theme's functions.php, and then tried the following:

``` php
var_dump( category_color_brightness( '#505050', 10 ) );
```

This returned `#363636`
I was getting valid values for other tests too... Until I tried more extreme values like this:

``` php
var_dump( category_color_brightness( '#505050', -200 ) );
```

in which case I got `#24d24d24d` so the only bug I could find was the fact that if the color is beyond the natural colors it will fail (working on a fix for that now).

But I was unable to get a value like the one you posted above...
What did you use as `$amount` to get that result?
